src/Aspire.Hosting/ApplicationModel/ResourceCommandAnnotation.cs

```diff
@@ -122,7 +122,19 @@ public static class CommandResults
     /// <summary>
     /// Produces a success result.
     /// </summary>
-    public static ExecuteCommandResult Success() => new ExecuteCommandResult { Success = true };
+    public static ExecuteCommandResult Success() => new() { Success = true };
+
+    /// <summary>
+    /// Produces an unsuccessful result with an error message.
+    /// </summary>
+    /// <param name="errorMessage">An optional error message.</param>
+    public static ExecuteCommandResult Failure(string? errorMessage = null) => new() { Success = false, ErrorMessage = errorMessage };
+
+    /// <summary>
+    /// Produces an unsuccessful result from an <see cref="Exception"/>. <see cref="Exception.Message"/> is used as the error message.
+    /// </summary>
+    /// <param name="exception">The exception to get the error message from.</param>
+    public static ExecuteCommandResult Failure(Exception exception) => Failure(exception.Message);
```

Comment:
Can we refactor the existing code that is creating `ExecuteCommandResult` instances to use these new methods? For example in the StressApp?

https://github.com/dotnet/aspire/blob/dc284ba1075b0f77ea445877a2167651fde2dafa/playground/Stress/Stress.AppHost/ResourceBuilderExtensions.cs#L53-L71

Actually, can that whole file go away now with this?